zkevm-circuits/src/util/word.rs

```diff
@@ -0,0 +1,214 @@
+//! Define generic Word type with utility functions
+// Naming Convesion
+// - Limbs: An EVN word is 256 bits. Limbs N means split 256 into N limb. For example, N = 4, each
+//   limb is 256/4 = 64 bits
+
+use eth_types::ToLittleEndian;
+use gadgets::util::{not, or, Expr};
+use halo2_proofs::{
+    circuit::{AssignedCell, Value},
+    halo2curves::FieldExt,
+    plonk::{Error, Expression},
+};
+use itertools::Itertools;
+
+use crate::evm_circuit::util::{from_bytes, CachedRegion, Cell};
+
+#[derive(Clone, Debug)]
+pub(crate) struct WordLimbs<T, const N: usize> {
```

Comment:
Please add a small documentation line about what is `N` here.  Also below there are some methods that use `N1` and `N2`, add also documentation about what those consts are.